src/main/java/com/epam/Main.java

```diff
@@ -1,24 +1,171 @@
 package com.epam;
 
-import com.epam.data.AccidentsDataLoader;
-import com.epam.data.RoadAccident;
-
+import java.io.FileWriter;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.LinkedList;
 import java.util.List;
+import java.util.concurrent.ArrayBlockingQueue;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.FutureTask;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVPrinter;
+
+import com.epam.data.RoadAccident;
+import com.epam.dataservice.AccidentBatchLoader;
+
 
 /**
  * Created by Tkachi on 2016/4/3.
  */
 public class Main {
+    
+    private static final int DEFAULT_THREAD_POOL_SIZE = 5;
+    private static final int DEFAULT_BATCH_SIZE = 40000;
+    private static final int DEFAULT_PRODUCER_SIZE = 2;
+    private static final int DEFAULT_CONSUMER_SIZE = 2;
+    private static final int DEFAULT_STORAGE_QUEUE_SIZE = 10;
+    private static final String OUTPUT_FILE_PATH_DAYTIME="src/main/resources/DaytimeAccidents.csv";
+    private static final String OUTPUT_FILE_PATH_NIGHTTIME="src/main/resources/NighttimeAccidents.csv";
+    private static CSVFormat csvFormat = CSVFormat.DEFAULT.withRecordSeparator('\n');
+ 
+    public static void main(String[] args) throws IOException, ExecutionException, InterruptedException {
+        String[] inputFilesList = args[0].split(",");
+        LinkedList<String> inputFileQueue = new LinkedList<String>();
+        inputFileQueue.addAll(Arrays.asList(inputFilesList));
+        
```

Comment:
I appreatiate how you made this application confugarable through the command line arguments. 
Just for your inforamtion - there are plenty of libs to help you with it. 
In case you need to build CLI in your project (I really hope you don't :smile: ) here is the library that I used once and it was quite nice. 
http://commons.apache.org/proper/commons-cli/usage.html